### ISSUE TITLE
There should be cell format compare option

### ISSUE THREAD

**Issue by vmkiili** [na-ka-na/ExcelCompare#47]
I noticed that ExcelCompare doesn't notice the difference between cells which are formatted in a different way. If one cell contains a date with time 00:00:00 and the other contains only the date, ExcelCompare thinks that they are the same.

**Comment by na-ka-na** [comment]
Yes this is documented in the limitations - Compares only cell "contents". Formatting, macros are currently ignored. I can add an option.
